### PR TITLE
Creating and Implementing Custom Container Component

### PR DESCRIPTION
**Pull request by calba5141114**
Creating a Custom Container as we begin to migrate away from bootstrap which will allow more freedom for future design decisions.

### CODE REVIEW

**Comment by calba5141114** [comment]
Screen Shot 2020-06-07 at 3 56 00 PM

**Comment by Drabblesaur** [comment]
Visually nothing has changed right?

**Comment by calba5141114** [comment]
It has changed a bit but not super noticeable. but at least if we don't like how it looks it is easily modifiable.

File: src/components/ui/Shared/Container.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import type { StyleDeclarationValue } from "aphrodite";
+import React from "react";
+import { StyleSheet, css } from "aphrodite";
+
+type ContainerType = "normal" | "fluid";
+
+type Props = {
+  readonly children: React.ReactNode;
```
Comment (calba5141114) [inline src/components/ui/Shared/Container.tsx:8]: React.ReactNode will basically be a type that will allow this component to be wrapped around other components or text or whatever can go inside a div

**Pull request merged**